os/kernel/semaphore/sem_tickwait.c

```diff
@@ -183,6 +183,11 @@ int sem_tickwait(FAR sem_t *sem, systime_t start, uint32_t delay)
 		/* Return the errno from sem_wait() */
 
 		ret = -get_errno();
+
+		/* Stop the watchdog timer as it's started before sem_wait */
+
+		wd_cancel(rtcb->waitdog);
```

Comment:
how about remove this and below line (goto errout_with_irqdisabled) to reduce code size? Is it not good for readability and posibility of adding new code?